go/libraries/doltcore/sqle/tables.go

```diff
@@ -179,7 +196,24 @@ func (t *DoltTable) GetIndexes(ctx *sql.Context) ([]sql.Index, error) {
 			tableSch:     sch,
 			unique:       index.IsUnique(),
 			comment:      index.Comment(),
+			generated:    false,
 		})
+		for i := 1; i < len(cols); i++ {
+			sqlIndexes = append(sqlIndexes, &doltIndex{
+				cols:         cols[:i],
+				db:           t.db,
+				id:           fmt.Sprintf("%s_PARTIAL_%d", index.Name(), i),
+				indexRowData: indexRowData,
+				indexSch:     index.Schema(),
+				table:        tbl,
+				tableData:    rowData,
+				tableName:    t.Name(),
+				tableSch:     sch,
+				unique:       index.IsUnique(),
```

Comment:
Probably better to say these are all non-unique... it shouldn't matter one way or another right now, but we definitely can't guarantee there will be at most one row for any equality lookup, which is the case for normal unique keys.

Reply:
Ohh that’s true. I’ll fix it.